kf_lib_data_ingest/network/oauth2.py

```diff
@@ -47,7 +47,8 @@ def get_service_token(provider_domain, audience, client_id, client_secret):
     logging.info(f'Fetching token from {oauth_token_url} to access '
                  f'{audience} resources')
 
-    response = requests.post(oauth_token_url, json=body)
+    response = requests_retry_session(connect=1).post(oauth_token_url,
```

Comment:
Is this another one of those mock problems?

Reply:
No I just didn't want it to be 10, I thought that was a lot? Idk what should it be